InnerEye/ML/configs/regression/LungRegression.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+
+from InnerEye.ML.configs.segmentation.Lung import Lung
+from InnerEye.ML.utils.split_dataset import DatasetSplits
+
+
+class LungRegression(Lung):
+    """
+    Model used for regression testing the Lung model. Uses the same data as the published model
+    but only runs for a small number of epochs and a smaller subset of the data. This is to ensure that
+    the training time is kept under 30 minutes to prevents slowing down PRs significantly.
+    """
+
+    def __init__(self) -> None:
+        super().__init__(
+            azure_dataset_id="lung",
+            train_batch_size=3,
+            num_epochs=25,
+            pl_deterministic=True,
+            test_crop_size=(64, 256, 256),
+        )
+
+    def get_model_train_test_dataset_splits(self, dataset_df: pd.DataFrame) -> DatasetSplits:
+        train = list(map(str, range(0, 10)))
```

Comment:
You can mayb use pl_limit_train/val/test_batches instead. not sure if those flags are available in innereye runner https://github.com/microsoft/hi-ml/blob/8b91cd0a7e2247a84059a90c4276b66f4f3837de/hi-ml/src/health_ml/deep_learning_config.py#L466